Cargo.lock

```diff
@@ -768,6 +768,7 @@ name = "hook-api"
 version = "0.1.0"
 dependencies = [
  "axum",
+ "chrono",
```

Comment:
That's interesting, news to me: https://github.com/chronotope/chrono/commits/main/

That said, I had to pull it into this crate for the PR, but `chrono` is already used elsewhere in the codebase. If you'd like use to switch then I'd rather do it as another PR since it will take a minute and seems unrelated.